plone/releaser/pypi.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
-import xmlrpclib
+
+try:
+    from xmlrpclib import ServerProxy
+except ImportError:
+    from xmlrpc.client import ServerProxy
 
 
 def get_users_with_release_rights(package_name):
-    client = xmlrpclib.ServerProxy('https://pypi.python.org/pypi')
+    client = ServerProxy('https://pypi.python.org/pypi')
```

Comment:
I am pretty sure this should be a different url: not on `pypi.python.org` but on `pypi.org`. But it seems to work still, so okay.

Reply:
According to warehouse's documentation: https://warehouse.pypa.io/api-reference/xml-rpc/

It should be: ``https://pypi.org/pypi``